lib/fairy/fairy.c

```diff
@@ -258,13 +259,34 @@ void Fairy_InitFile(FairyFileInfo* fileInfo, FILE* file) {
 
                     /* Ignore the leading "." */
                     if (strcmp(&shstrtab[currentSection.sh_name + 1], "text") == 0) {
-                        fileInfo->progBitsSizes[FAIRY_SECTION_TEXT] += ALIGN(currentSection.sh_size, 0x10);
+                        if (gUseElfAlignment) {
+                            // Ensure the next file will start at its correct alignment
```

Comment:
Also, the style I've been using is meant to be that line comments are temporary, although I've probably been inconsistent somewhere.

Reply:
Should I change this to something like this then?
```suggestion
                            /* Ensure the next file will start at its correct alignment */
```

Reply:
Yeah, since it's meant to be permanent.